index.js

```diff
@@ -50,8 +49,11 @@ function test(p, opts) {
   });
 }
 
+exports.test = test;
+
 module.exports = function (patterns, options) {
-  describe('eslint', function () {
+  options.contextName = options.contextName || 'eslint';
```

Comment:
Instead of changing the `contextName` of the options object itself, I'd prefer a new variable here:

```js
var contextName = (options && options.contextName) || 'eslint';
```

Also, it's not guaranteed that `options` will exist, so there needs to be a check before accessing the `contextName` property.  I notice now that there are no tests for that case (calling `lint` without any options), maybe you could add one?